packages/dx-react-chart/src/plugins/zoom-pan.tsx

```diff
@@ -51,37 +121,35 @@ class ZoomAndPanBase extends React.PureComponent<ZoomAndPanProps, ZoomAndPanStat
     };
   }
 
-  handleStart(zoomRegionKey: string) {
-    return (e: any) => {
-      this.offset = getOffset(e.currentTarget);
+  handleStart(zoomRegionKey: string, e: any) {
+    this.offset = getOffset(e.currentTarget);
       // Rectangle mode should be canceled if "zoomRegionKey" is released during mouse movevent or
       // not pressed when mouse is up. To do it access to "event" object is required in
       // "handleMouseMove" and "handleMouseUp".
       // TODO: Provide rectangle mode canceling.
-      if (isKeyPressed(e.nativeEvent, zoomRegionKey)) {
-        this.rectOrigin = [e.pageX - this.offset[0], e.pageY - this.offset[1]];
-      }
-      if (e.touches && e.touches.length === 2) {
-        this.multiTouchDelta = getDeltaForTouches(e.touches).delta;
-      }
-    };
+    if (isKeyPressed(e, zoomRegionKey)) {
+      this.rectOrigin = [e.pageX - this.offset[0], e.pageY - this.offset[1]];
+    }
+    if (e.touches && e.touches.length === 2) {
+      this.multiTouchDelta = getDeltaForTouches(e.touches).delta;
+    }
   }
 
-  handleTouchMove(scales: ScalesCache) {
-    return (e: any) => {
-      if (e.touches && e.touches.length === 2) {
-        const current = getDeltaForTouches(e.touches);
-        this.zoom(scales, current.delta - this.multiTouchDelta!, current.center);
-        this.multiTouchDelta = current.delta;
-      }
-    };
+  handleTouchMove(scales: ScalesCache, e: any) {
+    if (e.touches && e.touches.length === 2) {
+      const current = getDeltaForTouches(e.touches);
+      this.zoom(scales, current.delta - this.multiTouchDelta!, current.center);
+      this.multiTouchDelta = current.delta;
+    } else {
+      this.handleMouseMove(scales, { x: e.touches[0].clientX, y: e.touches[0].clientY });
+    }
+    e.preventDefault();
```

Comment:
It usually does first in callback (unless called only under certain conditions).